replicator/src/main/scala/com/evolutiongaming/kafka/journal/replicator/TopicReplicator.scala

```diff
@@ -267,9 +266,15 @@ private[journal] object TopicReplicator {
 
       for {
         consumer <- KafkaConsumerOf[F].apply[String, ByteVector](config1)
+        log      <- topicLoggerOf[F](topic).toResource
         metadata  = hostName.fold { Metadata.empty } { _.value }
-        commit    = TopicCommit(topic, metadata, consumer)
-        commit   <- TopicCommit.delayed(5.seconds, commit).toResource
+        commit <- TopicCommit.asyncPeriodic(
+          topic          = topic,
+          commitMetadata = metadata,
+          commitPeriod   = 5.seconds,
```

Comment:
these `5.seconds` are not documented, right?

shall we update ScalaDoc or expose it as settable argument with default value?